develop/devguide/Connector/LogicQActions.md

```diff
@@ -119,6 +119,9 @@ In the example above, the Run method of the QAction class is defined as a static
 
 It is also possible to define static fields in the QAction class. As only one copy of a static member exists, regardless of how many instances of the class are created, static fields are also shared between all the elements executing this protocol and persist as long as the SLScripting process is running.
 
+> [!NOTE]
+> Static fields are only shared between elements with the same DataMiner version.
+
```

Comment:
@Robin-Spruytte I have indeed changed this into a note. Is this OK to merge?

Reply:
Hi Edith, I meant protocol version, not DataMiner version. There was some confusion whether the fields are shared between same protocols if the protocol version was different.

Reply:
Changed it to protocol version!